src/string.cpp

```diff
@@ -411,6 +411,30 @@ gb_internal String concatenate4_strings(gbAllocator a, String const &x, String c
 	return make_string(data, len);
 }
 
+#if defined(GB_SYSTEM_WINDOWS)
+gb_internal String escape_char(gbAllocator a, String s, char cte) {
+	isize buf_len = s.len;
+	isize cte_count = 0;
+	for (isize j = 0; j < s.len; j++)
+		if (s.text[j] == cte)
+			cte_count++;
```

Comment:
Please use braces. That's our general style where possible.

Reply:
braces added